src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1791,6 +1791,9 @@ internal bool CanDelete(object parameters)
         public void SaveImage(object parameters)
         {
             OnRequestSaveImage(this, new ImageSaveEventArgs(parameters.ToString()));
+
+            Dynamo.Logging.Analytics.CreateCommandEvent("SaveImageCapture",
+                "Nodes", CurrentSpace.Nodes.Count());
```

Comment:
Consider "NodeCounts" instead of "Nodes" to make it clear that it's just a number